include/tcpdf_cmssignature.php

```diff
@@ -0,0 +1,545 @@
+<?php
+/*
+// File name   : tcpdf_cmssignature.php
+// Version     : 1.0.1
+// Begin       : 2023-05-25
+// Last Update : 2023-05-25
+// Author      : Hida - https://github.com/hidasw
+// License     : GNU GPLv3
+*/
+/**
+ * @class tcpdf_cms_signature
+ * Manage CMS Signature for TCPDF.
+ * @version 1.0.001
+ * @author M Hida
+ */
+class tcpdf_cms_signature {
+	/**
+	 * Catch error.
+	 * @public
+	 */
+	public $errorMsg;
+	
+	/**
+	 * result value of pkcs7 EncryptedDigest.
+	 * @public
+	 */
+	public string $pkcs7_EncryptedDigest;
```

Comment:
Typed properties are only allowed since PHP 7.4. As this library require "php": ">=5.3.0", this may cause problems in php versions under 7.4.